### PR TITLE
add blog

### PR DESCRIPTION
**Pull request by manthanank**

### CODE REVIEW

File: data.js [hunk 1]
```diff
@@ -4368,5 +4368,33 @@ module.exports = [
...
     twitter: '@mahdix',
     tags: ['Java', 'SaaS', 'Kotlin', 'Linux'],
-  }
+  },
+  {
+    name: "Manthan Ank",
+    description: "I'm Front-End Developer, I write tutorials related to Web Development.",
+    url: "https://dev.to/manthanank",
+    twitter: "@manthanank",
+    tags: [
```
Comment (AbbeyIT) [inline data.js:4377]: Hi @manthanank ! Kindly lessen the tags to at least **10** tags only. Feel free to just commit here again in the PR. Lmk if you need any help.

**Review by AbbeyIT — CHANGES_REQUESTED** [review]

File: data.js (hunk 1 above)
Comment (manthanank) [reply data.js:4357]: removed few tags

**Comment by manthanank** [comment]
Requested changes made

**Review by AbbeyIT — APPROVED** [review]
LGTM! Thank you for your contribution.

**Pull request merged**